.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
you can remove this file as it has no content

Reply:
done